### ISSUE TITLE
Instructions for creating authors are not complete

### ISSUE THREAD

**Issue by georgik** [espressif/developer-portal#109]
The documnet https://developer.espressif.com/pages/contribution-guide/content-writing-workflow/#add-youself-as-an-author is not accurate. It does not explain expected content of index.md.

The minimal version should be:
```
---
title: Juraj Michálek
---
```

The problem is that if you put there just `name:` instead of title, then the user name won't be visible at Authors page.

**Comment by FBEZ** [comment]
I agree with Juraj, I had to create the first content a few weeks back and even if it's possible to look around files and folder and figure out how to do, it can be easier. 

I would change [these lines in the article](https://github.com/espressif/developer-portal/blob/e4261cc219225074f7e7c44a0510b4f2c7745e18/content/pages/contribution-guide/writing-content/index.md?plain=1#L45-L58) to make it a little more verbose.


Below an example. If you like, I could create a PR.

---

In other cases, add yourself as an author. 
You need to 
1. Create your author entry 
2. Add your personal data
3. (optional) Add your picture. 
4. Reference yourself as an author in the article

__Create your author entry__

You need to create your personal page at `content/authors/<author-name>/_index.md`
`hugo create content/authors/<author-name>/_index.md`

Open the file and enter the 
```
---
title: "John Doe"
---
```

__Add author data__

The data is stored as json at `data/authors/<author-name>.json`. Create the file and add 
```json
{
    "name": "John Doe",
    "image": "img/authors/john-doe.webp",
    "bio": "Espressif enthusiast",
    "social": [
        { "linkedin": "https://www.linkedin.com/in/..." },
        { "twitter": "https://x.com/..." }
    ]
}
```

__Add your picture__

If you like to add your picture (shown at the beginning of the article) you can add a [webp](https://it.wikipedia.org/wiki/WebP) portrait at `assets/img/authors/john-doe.webp`


__Reference yourself as an author__

In your article's YAML front-matter, add the following lines

  ```yaml
  showAuthor: false         # Hide default Espressif author
  authors:
    - "<author-name>"        # List your name(s)
  ```

---

(BTW The link is now changed to https://developer.espressif.com/pages/contribution-guide/writing-content/#add-youself-as-an-author. )
